lib/pages/contacts_tab/contacts_tab_body_view.dart

```diff
@@ -487,3 +499,65 @@ class _Contact extends StatelessWidget {
     );
   }
 }
+
+class _SilverUnknownContact extends StatelessWidget {
+  final ContactsTabController controller;
+
+  const _SilverUnknownContact({required this.controller});
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+      valueListenable: controller.presentationRecentContactNotifier,
+      builder: (_, recentContacts, __) {
+        return ValueListenableBuilder(
+          valueListenable: controller.presentationContactNotifier,
+          builder: (___, presentationContactState, ____) {
+            return ValueListenableBuilder(
+              valueListenable: controller.presentationPhonebookContactNotifier,
+              builder: (context, phonebookContactState, child) {
```

Comment:
For me, this logic terrible. 

- What is exactly you want to handle?
- If used like this, all failure and success cases will go here. Is it really necessary?
- Have you thought carefully about the issue and solution?
- What is the root cause here?